addons/stock/models/stock_picking.py

```diff
@@ -635,7 +635,12 @@ def action_done(self):
         self.write({'date_done': fields.Datetime.now()})
         return True
 
-    do_transfer = action_done #TODO:replace later
+    # Backward compatibility
+    # Problem with fixed reference to a function:
+    # it doesn't allow for overriding action_done() through do_transfer
+    # get rid of me in master (and make me private ?)
+    def do_transfer(self):
```

Comment:
@sle-odoo 
Overkill much ?

Reply:
@sle-odoo we should also check at this one: https://github.com/odoo/enterprise/blob/11.0/quality/models/stock_picking.py#L64
and this one: https://github.com/odoo/enterprise/blob/11.0/sale_ebay/models/stock_picking.py#L11

Reply:
I made changes in PR https://github.com/odoo/enterprise/pull/1746, waiting them green to merge it.